### PR TITLE
remnote: 1.13.52 -> 1.14.13

### PR DESCRIPTION
**Pull request by chewblacka**
## Description of changes
https://feedback.remnote.com/changelog

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by matthiasbeyer — DISMISSED** [review]
There's a lot of changes in this patch that are not a simple version update.

Especially the removal of one maintainer of the package.

I'd like to have the commit split up into several commits, if you don't mind.

**Comment by chewblacka** [comment]
> There's a lot of changes in this patch that are not a simple version update.
> 
> Especially the removal of one maintainer of the package.
> 
> I'd like to have the commit split up into several commits, if you don't mind.

Have split into 3 commits. Hopefully that is better. Many thanks.

File: pkgs/applications/misc/remnote/default.nix [hunk 1]
```diff
@@ -29,22 +32,25 @@ in
...
+    install -Dm444 ${icon} $out/share/pixmaps/remnote.png
 
     runHook postInstall
   '';
+
   meta = with lib; {
     description = "A note-taking application focused on learning and productivity";
     homepage = "https://remnote.com/";
-    maintainers = with maintainers; [ max-niederman chewblacka ];
+    maintainers = with maintainers; [ chewblacka ];
```
Comment (0x4A6F) [inline pkgs/applications/misc/remnote/default.nix:53]: Why?

Comment (chewblacka) [reply pkgs/applications/misc/remnote/default.nix:53]: Maintainer hasn't been active on this package in at least the last 6 months. Keeps auto-adding him as a reviewer and not giving any other suggestions.

Comment (chewblacka) [reply pkgs/applications/misc/remnote/default.nix:53]: > Why?
Maintainer hasn't been active on this package in at least ~~the last 6 months~~.

Make that in nearly 2 years (May 2022).

Comment (0x4A6F) [reply pkgs/applications/misc/remnote/default.nix:53]: Please split out this change into a separate PR.
There should be at least a distinct maintainer timeout on a removal, that isn't bound to a package update (defering the package update needlesly).

Comment (chewblacka) [reply pkgs/applications/misc/remnote/default.nix]: > Please split out this change into a separate PR. There should be at least a distinct maintainer timeout on a removal, that isn't bound to a package update (defering the package update needlesly).

Done.

**Review by 0x4A6F — APPROVED** [review]

**Review by eclairevoyant — APPROVED** [review]

**Comment by chewblacka** [comment]
Can this be merged? Many thanks.

**Pull request merged**